feat: Multi Dev Portal (#7561)
Kong/docs.konghq.com: app/konnect/dev-portal/applications/enable-app-reg.md

> 

Author: @Guaris I made https://konghq.atlassian.net/browse/DOCU-3951 to handle the couple misc feedback you had here

## Prerequisites

- [An API product that is versioned and published to a {{site.konnect_short_name}} Dev Portal](/konnect/dev-portal/publish-service) so that it appears in the catalog.

- If you are using [OpenID Connect](#oidc-flow) for your authorization:

  - Set up your application, claims, and scopes in your OpenID identity provider. Refer to your IdP/OP provider's documentation for instructions.

> 
>   - Edit the **Reference ID** field in the Dev Portal
>     [Update Application](/konnect/dev-portal/applications/dev-apps#edit-my-app)

Reviewer: this step is from the dev side, which doesn't make sense because the rest of the instructions (and the page) are from the admin side. I suggest removing this sentence
```suggestion
```

>     dialog to match to your third-party OAuth2 claim. <!-- is this on the admin side of Dev Portal UI, or dev side?-->

Reviewer: this is on the dev portal side

{:.note}
> **Note:** Generally, it's only recommended for an API product version to be linked to a Gateway service. However, for app registration to work, the link between API product version and a Gateway service is required. 

## Enable app registration with key authentication {#key-auth-flow}

1. In {% konnect_icon api-product %} **API Products**, select a service.

1. Click **Product Versions** and select a product version.

  {:.note}
  > **Note:** If the API product version is in the `default` control plane group, it will
  instead receive read-only entries for the `acl` and `key-auth` plugins to provide
  support for {{site.base_gateway}} versions less than 3.0.

> 1. From the **Dev Portals** tab, click **Publish API**.

Reviewer: ```suggestion
1. From the **Dev Portals** tab, click **Publish to Dev Portals**.
```

> 1. Enable **Require Authentication** and select **key-auth** for your auth strategy for each Dev Portal.

Reviewer: ```suggestion
1. Enable authentication by toggling on **Require Authentication**. If your API Product is already published to a Dev Portal, select **key-auth** in the Auth Strategy drop down for that Dev Portal. If your API Product is not yet published click **+ Dev Portal**, select the target Dev Portal in the dropdown, and then select **Key Auth** in the Auth Strategy dropdown selection.
```

> 1. Enable **Accept application registrations** for each Dev Portal.

Reviewer: ```suggestion
1. Enable **Accept application registrations** for each Dev Portal to allow developers to register their applications to consume this API.
```

5. Optional: Click the [**Auto Approve**](/konnect/dev-portal/access-and-approval/auto-approve-devs-apps/) checkbox if you want to automatically approve all application registration requests.

1. Click **Save**.

> ## Enable app registration with OpenID Connect {#oidc-flow}

Reviewer: need to update the steps for oidc too
